lib/surveyor/engine.rb

```diff
@@ -1,8 +1,6 @@
 require 'rails'
 require 'surveyor'
 require 'simple_form'
```

Comment:
Can we remove this line (and the Gem dependency)? If we need this in our app -- we should just add it.